webapp/store/snap_details_views.py

```diff
@@ -276,3 +277,27 @@ def snap_details_case_sensitive(snap_name):
         return flask.redirect(
             flask.url_for(".snap_details", snap_name=snap_name.lower())
         )
+
+    @store.route('/<regex("' + snap_regex + '"):snap_name>/badge')
+    def snap_details_badge(snap_name):
+        context = _get_context_snap_details(snap_name)
+
+        snap_link = "https://snapcraft.io/" + context["package_name"]
```

Comment:
You could use `flask.request.url_root` instead of `https://snapcraft.io/`, just so it works when we are testing in staging.

Reply:
Sure didn't know that, thanks.